mula/scheduler/connectors/services/octopoes.py

```diff
@@ -10,8 +10,15 @@ class Octopoes(HTTPService):
     name = "octopoes"
     health_endpoint = None
 
-    def __init__(self, host: str, source: str, orgs: List[Organisation]):
+    def __init__(
+        self,
+        host: str,
+        source: str,
+        orgs: List[Organisation],
+        request_timeout: int = 10,
+    ):
         self.orgs: List[Organisation] = orgs
+        self.timeout = request_timeout
```

Comment:
Why not just pass this argument to the initializer below? (not tested) By just reading the code I can see that this argument is overridden by the parent's initializer to its default value (5)

Reply:
Yes, you are correct. This is an oversight